js/lib/comments.js

```diff
@@ -113,6 +113,10 @@ elgg.comments.init = function() {
 		dc.toggleEdit();
 		return false;
 	});
+
+	// Highlight a comment if its id is found from the URL
+	var commentId = elgg.getSelectorFromUrlFragment(document.URL);
+	$(commentId).addClass('elgg-highlight').delay(1000).removeClass('elgg-highlight', {duration: 1000});
```

Comment:
I think this only works with jQueryUI present. So you might as well use
$.highlight()
On Mar 9, 2015 5:37 PM, "Evan Winslow" <notifications@github.com> wrote:

> In js/lib/comments.js
> <https://github.com/Elgg/Elgg/pull/8050#discussion_r26051581>:
>
> > @@ -113,6 +113,10 @@ elgg.comments.init = function() {
> >  		dc.toggleEdit();
> >  		return false;
> >  	});
> > +
> > +	// Highlight a comment if its id is found from the URL
> > +	var commentId = elgg.getSelectorFromUrlFragment(document.URL);
> > +	$(commentId).addClass('elgg-highlight').delay(1000).removeClass('elgg-highlight', {duration: 1000});
>
> What does it mean to remove a class with a duration of 1000ms?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8050/files#r26051581>.
>